.github/workflows/helm.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run this workflow on new tags
+name: Helm
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  update-helm-charts:
+    name: Update gh-pages helm repo index
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Helm
+      uses: azure/setup-helm@v4.2.0
+      with:
+        version: 3.14.4
+
+    - name: Check out repo
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Update repo index
+      run: |
+        git config user.name "Github Actions"
+        git config user.email "no-reply@github.com"
+        ./hack/update-helm-index.sh  ${{ join(github.event.release.assets.*.browser_download_url, ' ') }}
```

Comment:
Can you move the update-helm-index.sh command argument to a variable so that it's more readable?

Reply:
Can we run these scripts as makefile targets instead?

Reply:
Since I want to take arguments, and let GitHub to build the argument from secrets, I prefer to leave it as a script